distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1287,7 +1287,7 @@ type Integer
          Parse the text "20220216" into an integer number.
 
             Integer.parse "20220216"
-    parse text:Text (radix=10:Integer) -> Integer ! Number_Parse_Error = integer_parse text radix
+    parse text:Text (radix:Integer = 10) -> Integer ! Number_Parse_Error = integer_parse text radix
```

Comment:
Isn't this a flaw of the Enso language?

Personally I never know whether to type the former or the latter variant or some other. I try few and whichever compiles I keep. I wasn't aware there could be a difference between `r=10:Int` and `r:Int=10`!

Does there have to be a difference? CCing @kazcw.

Reply:
Yes, that's how it behaves. However we could also parse `radix=10:Integer` as
- `radix` argument name
- `10` default value
- and type ascription `Integer` 

and interpret it as `radix:Integer = 10` probably (if the AST isn't against it completely).

Reply:
We _could_ but I don't know of any language that allows to reorder types and default arguments like this. Won't that make the language unnecessarily complicated?

Even if we change the parser like you suggest, we still have the case of `radix=(10:Integer)`. With the parentheses I think we still should treat it as bound to the expression, not the argument, right?